docs/03-code-internals/19-service-objects.md

```diff
@@ -0,0 +1,601 @@
+---
+title: Using service objects at Discourse
```

Comment:
```suggestion
title: Using service objects in Discourse
```

Reply:
I was thinking of using services at Discourse the org when writing that, actually. You still think that would be better to replace it by “in”? (I guess using “in” is to say Discourse the product/code base?)

Reply:
Yeah, I think "in" is more inclusive as these docs are published on meta and can also enable contributions from the community.

Reply:
Done! :slightly_smiling_face: